packages/storefront-sdk/package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "@nacelle/storefront-sdk-v2",
+	"version": "2.0.0",
```

Comment:
I'll create a new ticket to capture the task of entering changesets prerelease mode.